x/globalfee/client/cli/query.go

```diff
@@ -24,19 +24,20 @@ func GetQueryCmd() *cobra.Command {
 
 func GetCmdShowMinimumGasPrices() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "minimum-gas-prices",
-		Short:   "Show minimum gas prices",
-		Long:    "Show all minimum gas prices",
-		Aliases: []string{"min"},
-		Args:    cobra.ExactArgs(0),
+		Use:   "params",
+		Short: "Show globalfee params",
+		// todo check here
+		Long: "Show globalfee requirement: minimum_gas_prices, bypass_min_fee_msg_types, max_total_bypass_minFee_msg_gas_usage",
+		//	Aliases: []string{"min"},
+		Args: cobra.ExactArgs(0),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			clientCtx, err := client.GetClientQueryContext(cmd)
 			if err != nil {
 				return err
 			}
 
 			queryClient := types.NewQueryClient(clientCtx)
-			res, err := queryClient.MinimumGasPrices(cmd.Context(), &types.QueryMinimumGasPricesRequest{})
+			res, err := queryClient.Params(cmd.Context(), &types.QueryParamsRequest{})
```

Comment:
Don't we have a test for this?

Reply:
Good point. We don't at the moment because it'd require defining a new TestSuite afaiu.
Maybe we can open an issue for this.